api/http/server.go

```diff
@@ -51,28 +53,29 @@ import (
-	getFileNotFound = metrics.NewRegisteredCounter("api/http/get/file/notfound", nil)
-	getFileFail     = metrics.NewRegisteredCounter("api/http/get/file/fail", nil)
-	getListCount    = metrics.NewRegisteredCounter("api/http/get/list/count", nil)
-	getListFail     = metrics.NewRegisteredCounter("api/http/get/list/fail", nil)
-	getTagCount     = metrics.NewRegisteredCounter("api/http/get/tag/count", nil)
-	getTagNotFound  = metrics.NewRegisteredCounter("api/http/get/tag/notfound", nil)
-	getTagFail      = metrics.NewRegisteredCounter("api/http/get/tag/fail", nil)
-	getPinCount     = metrics.NewRegisteredCounter("api/http/get/pin/count", nil)
-	getPinFail      = metrics.NewRegisteredCounter("api/http/get/pin/fail", nil)
-	postPinCount    = metrics.NewRegisteredCounter("api/http/post/pin/count", nil)
-	postPinFail     = metrics.NewRegisteredCounter("api/http/post/pin/fail", nil)
-	deletePinCount  = metrics.NewRegisteredCounter("api/http/delete/pin/count", nil)
-	deletePinFail   = metrics.NewRegisteredCounter("api/http/delete/pin/fail", nil)
+	postRawCount       = metrics.NewRegisteredCounter("api/http/post/raw/count", nil)
+	postRawFail        = metrics.NewRegisteredCounter("api/http/post/raw/fail", nil)
+	postFilesCount     = metrics.NewRegisteredCounter("api/http/post/files/count", nil)
+	postFilesFail      = metrics.NewRegisteredCounter("api/http/post/files/fail", nil)
+	deleteCount        = metrics.NewRegisteredCounter("api/http/delete/count", nil)
+	deleteFail         = metrics.NewRegisteredCounter("api/http/delete/fail", nil)
+	getCount           = metrics.NewRegisteredCounter("api/http/get/count", nil)
+	getFail            = metrics.NewRegisteredCounter("api/http/get/fail", nil)
+	getFileCount       = metrics.NewRegisteredCounter("api/http/get/file/count", nil)
+	getFileNotFound    = metrics.NewRegisteredCounter("api/http/get/file/notfound", nil)
+	getFileFail        = metrics.NewRegisteredCounter("api/http/get/file/fail", nil)
+	getListCount       = metrics.NewRegisteredCounter("api/http/get/list/count", nil)
+	getListFail        = metrics.NewRegisteredCounter("api/http/get/list/fail", nil)
+	getTagCount        = metrics.NewRegisteredCounter("api/http/get/tag/count", nil)
+	getTagNotFound     = metrics.NewRegisteredCounter("api/http/get/tag/notfound", nil)
+	getTagFail         = metrics.NewRegisteredCounter("api/http/get/tag/fail", nil)
+	getPinCount        = metrics.NewRegisteredCounter("api/http/get/pin/count", nil)
+	getPinFail         = metrics.NewRegisteredCounter("api/http/get/pin/fail", nil)
+	postPinCount       = metrics.NewRegisteredCounter("api/http/post/pin/count", nil)
+	postPinFail        = metrics.NewRegisteredCounter("api/http/post/pin/fail", nil)
+	deletePinCount     = metrics.NewRegisteredCounter("api/http/delete/pin/count", nil)
+	deletePinFail      = metrics.NewRegisteredCounter("api/http/delete/pin/fail", nil)
+	errRecoveryAttempt = errors.New("recovery was initiated")
```

Comment:
this looks a bit out of place here with these metrics. 

not sure where else to place it though, `api` seems like a big place :man_shrugging:

Reply:
I was not sure where to leave it, but as @zelig mentioned moved it to netstore.
Done